### PR TITLE
fix: Fix broken links in ADOPTERS.md and graphviz.js

### PR DESCRIPTION
**Pull request by dol**
## 📑 Proposed changes

Fix broken links. The later was responsible for not loading the graphviz library and left the graphviz parts in the page unrendered.

In the later stage of the project it might be useful to add a link checker similar to the linter check for pull requests. Tools like https://github.com/lycheeverse/lychee-action can detect broken links in markdown files.
This could hinder the contributions like:
- https://github.com/aep-dev/aep.dev/blob/f256e1d4db14a338b51a8cf58aa1224a6374cd54/aep/general/0155/aep.md.j2#L91 is not yet ported, but was already linked.
- Merge of improvement https://github.com/aep-dev/aep.dev/blob/eb889fb5905fe0c64878c7b72cdca44fc5bcf03f/aep/general/0151/aep.md.j2#L161 was done before it's dependent PR #8 was merged.

image

## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [x] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [ ] Migrated from google.aip.dev

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [x] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [x] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ name to this list (in alphabetical order).
...
-| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aep.dev) and [site generator](https://github.com/aep-dev/site-generator). |
+| [Google](https://about.google/)    | [@toumorokoshi](https://github.com/toumorokoshi) | Google maintains a fork at [google.aip.dev](http://google.aip.dev) which is adhered to for all APIs. Google also runs the [api-linter](https://linter.aip.dev/) across the company internally.                                                 |
+| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aip.dev) and [site generator](https://github.com/aep-dev/site-generator). |
```
Comment (toumorokoshi) [inline ADOPTERS.md:12]:
```suggestion
| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aip.dev) and [site generator](https://github.com/aep-dev/site-generator). |
```

Actually Google is not affiliated with the AEPs (at least not officially) - we can remove this line for now.

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
Good catch! One quick issue around the Google change - google is not affiliated with AEPs.

All else looks good!

File: ADOPTERS.md (hunk 1 above)
Comment (dol) [reply ADOPTERS.md]: I removed the section entirely.

**Review by toumorokoshi — APPROVED** [review]
Great, thanks!

**Comment by toumorokoshi** [comment]
Oh! almost. @dol can you run prettier to fix? see https://aep.dev/contributing#formatting

**Comment by dol** [comment]
> Oh! almost. @dol can you run prettier to fix? see https://aep.dev/contributing#formatting

I always forget this. Done now.

**Comment by toumorokoshi** [comment]
Thank you!

**Pull request merged**